pkgs/development/tools/database/atlas/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, git
+}:
+
+buildGoModule rec {
+  pname = "atlas";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "ariga";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-sC0RkRUtOkmQ07A7WVU9Tg29Ffk6H26z7FdtnMgatc8=";
+  };
+
+  vendorSha256 = "sha256-XjmXMj9EaRuqSqAgZdvi51gy2gMbwYk6/J40qbQH8CQ";
+
+  proxyVendor = true;
+
+  # subPackages = [ "cmd/atlas"] does not work
+  preBuild = ''
+    cd cmd/atlas
+  '';
```

Comment:
```suggestion
  modRoot = "cmd/atlas";
```

Reply:
yeah, this is failing because there is a submodule with a workspace contained in cmd/atlas, so you must situate the module root here, not at the repo root